src/meta_memcache/base/base_cache_pool.py

```diff
@@ -205,11 +208,15 @@ def _conn_recv_response(
             return Success(flags=set([Flag.NOREPLY]))
         result = conn.get_response()
         if isinstance(result, Value):
-            # TODO: Confirm this works for empty values!
             data = conn.get_value(result.size)
             if result.size > 0:
                 encoding_id = result.int_flags.get(IntFlag.CLIENT_FLAG, 0)
-                result.value = self._serializer.unserialize(data, encoding_id)
+                try:
+                    result.value = self._serializer.unserialize(data, encoding_id)
+                except Exception:
+                    _log.exception("Error unserializing value")
```

Comment:
TIL this exists, I've always been doing `.error` with `exc_info=True` which I think is the same thing.